src/node/Params.hs

```diff
@@ -100,6 +101,7 @@ getNodeParams args@Args {..} systemStart = do
             then genesisUtxo Nothing (devAddrDistr devStakeDistr)
             else genesisUtxo (Just genesisProdBootStakeholders)
                              genesisProdAddrDistribution
+    logInfo $ sformat ("npCustomUtxo: "%shown) npCustomUtxo
```

Comment:
1. There is `utxoF` in `Pos.Txp.Toil.Types` which formats it better than `shown`.
2. Sometimes this utxo can be very big and I don't want it to take hundreds or thousands of lines in the output. I would print it only if it is relatively small (at most 25 for example). If it's big, I would print only the size of it. If it's really needed, it's better to dump it to a file.